apps/style/craft/style.scss

```diff
@@ -400,7 +405,7 @@ $height-to-width: 434 / 477;
 
 #craft-popup-connect {
   @include reset-dialog;
-  background-color: #3C3C3C;
+  background-color: $almost-very-dark-gray;
```

Comment:
I'm not sure about clearer, but I think it's useful in the sense that you end up with this block near the top of the file:
```
$darkest-gray: #333;
$very-dark-gray: #3b3b3b;
$almost-very-dark-gray: #3c3c3c;
```
It amplifies a couple of style-smells: That we are using three near-identical colors, and that we don't have a semantic name for any of them (where colors `$button-text-white` and `$header-text-yellow` are more obviously useful because it's clear when/where to use them in the future).  I'm not bothering to fix that here because I'd like this to be a pure refactor, but I think it's more likely to get cleaned up next time somebody works on these colors if they're grouped like that.